### PR TITLE
Adding Dockerfile to build a testing container

### PR DESCRIPTION
**Pull request by coolo**
I developed this as mean to test the full-stack test without fearing
about my system - and being closer to travis-ci environment

### CODE REVIEW

File: docker/testing/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Docker file to test full-stack.t in a travis like environment
+FROM ubuntu:12.04
+run apt-get -y update
+run apt-get -y install python-software-properties
+run add-apt-repository -y ppa:kubuntu-ppa/backports
+run apt-get -y update
+run apt-get -y install libdbus-1-dev libssh2-1-dev libopencv-dev libtheora-dev libcv-dev libhighgui-dev tesseract-ocr libsndfile1-dev libfftw3-dev qemu-system automake libtool libxml2-dev libgmp3-dev ruby-sass git libssl-dev libgcrypt11-dev curl wget
+run curl http://stephan.kulow.org/phantomjs-2.1.1-linux-x86_64.tar.bz2 | tar xvjf -
```
Comment (coolo) [inline docker/testing/Dockerfile]: in case you wonder: I just mirrored it as the official download link is very unreliable

**Comment by okurz** [comment]
> without fearing about my system …

but then normally I end up with docker filling up my harddisk. I guess you made the same experience ;-)

**Comment by coolo** [comment]
Actually btrfs snapshots were the real killer :)

**Comment by okurz** [comment]
Looks like a nice approach. All this magic version referencing looks hacky to me and will probably outdate quite soonish. Anyway, go ahead.

**Comment by coolo** [comment]
the perl version? that's the one of sle12 and will stay the reference for quite a while unfortunately :)

**Pull request merged**